docs/framework/guides/contributing/git-commit-message-convention.md

```diff
@@ -172,7 +172,7 @@ Changelog
 
 ## [1.0.0](https://github.com/ckeditor/ckeditor5/compare/v1.0.0...v0.0.1) (2017-01-04)
 
-### MAJOR BREAKING CHANGES [ℹ️](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/versioning-policy.html#major-and-minor-breaking-changes)
+### MAJOR BREAKING CHANGES [ℹ️](https://ckeditor.com/docs/ckeditor5/latest/support/versioning-policy.html#major-and-minor-breaking-changes)
```

Comment:
It would be good to update source scripts too: https://github.com/ckeditor/ckeditor5-dev/blob/8ebb83bd777a4ad05c9c421d7246fde17abba1c7/packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelogformonorepository.js#L27-L28.